Documentation/operations/upgrade.rst

```diff
@@ -398,6 +398,8 @@ Annotations:
   ``CiliumLoadBalancerIPPool.spec.blocks``. As of v1.15 both fields have the same behavior. The
   ``cidrs`` field will be removed in v1.16. Please update your IP pool configurations to use
   ``blocks`` instead of ``cidrs`` before upgrading.
+* The use of per-tunnel IPsec keys is mandatory, via the use of the ``+`` sign
+  in the secret. See the :ref:`encryption_ipsec` guide for more information.
```

Comment:
Suggestion, non-blocking:

```suggestion
* For IPsec, the use of per-tunnel keys is mandatory, via the use of the ``+`` sign
  in the secret. See the :ref:`encryption_ipsec` guide for more information.
```

(They're only mandatory if IPsec encryption is enabled. It should be rather obvious, but :shrug:)

Reply:
I think it's worth a change, mostly because it's faster for readers to know in what context this upgrade note (among the many other notes) matters.